While Building Stellar-Core in Ubuntu using Windows Subsystem for Linux

# Issue type
   #006a4f9

# Version
 Stellar Core 10.3.0

# Environment and setup
   Ubuntu 18.04.1 LTS

# Issue description
 When I am running **make check** in terminal getting test failed at one point. Please find below screenshot for your reference what exactly i am getting. Please anyone help me regarding this issue. 

`/home/stellar-core/src/test/selftest-parallel: line 69: parallel: command not found
FAIL: test/selftest-pg
==================
1 of 1 test failed
==================
Makefile:2130: recipe for target 'check-TESTS' failed
make[3]: *** [check-TESTS] Error 1
make[3]: Leaving directory '/home/stellar-core/src'
Makefile:2253: recipe for target 'check-am' failed
make[2]: *** [check-am] Error 2
make[2]: Leaving directory '/home/stellar-core/src'
Makefile:2255: recipe for target 'check' failed
make[1]: *** [check] Error 2
make[1]: Leaving directory '/home/stellar-core/src'
Makefile:569: recipe for target 'check-recursive' failed
make: *** [check-recursive] Error 1`

![Capture3](https://user-images.githubusercontent.com/29983995/55003007-8482de00-4ffd-11e9-8c5e-0121cc2d2417.PNG)

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -61,7 +61,7 @@ After installing packages, head to [building with clang and libc++](#building-wi
 
 #### Installing packages
     # common packages
-    sudo apt-get install git build-essential pkg-config autoconf automake libtool bison flex libpq-dev
+    sudo apt-get install git build-essential pkg-config autoconf automake libtool bison flex libpq-dev parallel
     # if using clang
     sudo apt-get install clang-5.0
     # clang with libstdc++

